docs/podman-network-create.1.md

```diff
@@ -0,0 +1,70 @@
+% podman-network-create(1)
+
+## NAME
+podman\-network-create- Create a Podman CNI network
+
+## SYNOPSIS
+**podman network create**  [*options*] name
+
+## DESCRIPTION
+Create a CNI configuration for use with Podman. At the time of this writing, the only network
+type that can be created is a *bridge* network.
+
+If no options are provided, Podman will assign a free subnet and name for your network.
+
+Upon completion of creating the network, Podman will display the path to the newly added network file.
+
+## OPTIONS
+**-d**, , **--driver**
+
+Driver to manage the network (default "bridge")
+
+**--gateway**
+
+Define an IPv4 or IPv6 gateway for the subnet. If you want to provide a gateway address, you must also provide a
```

Comment:
I thought we were only supporting IPV4?

Reply:
IPv6 will come ... but we can clean this up